app/adapters/application.js

```diff
@@ -1,6 +1,19 @@
-import HalAdapter from "ember-data-hal-9000/adapter";
-import config from '../config/environment';
+import HalAdapter from 'ember-data-hal-9000/adapter';
+import storage from '../utils/storage';
+import config from "../config/environment";
 
 export default HalAdapter.extend({
-  host: config.apiURL
+  host: config.apiHost,
+
+  headers: function(){
+    let headers = {};
+    let token = storage.read(config.authTokenKey);
+    if (token) {
+      // For rails token authorization
+      headers['Authorization'] =
+        `Token token=${token.token}, email=${token.email}`;
```

Comment:
this is an [ES6 template string](http://updates.html5rocks.com/2015/01/ES6-Template-Strings). We can write ES6 code in ember-cli now due to the 6to5 plugin.